src/redux/reducers/index.js

```diff
@@ -3,11 +3,15 @@ import ClustersReducer from './clustersReducer';
 import AddClusterReducer from './addClusterReducer';
 import user from './user';
 import ClusterResourcesReducer from './ClusterResourcesReducer';
+import ClustersReducer from './clustersReducer';
```

Comment:
```suggestion
```
This line causes a run time error because your importing ClustersReducer twice

Reply:
@Mubangizi thank you for pointing that out